go-controller/pkg/cni/helper_linux.go

```diff
@@ -246,10 +246,18 @@ func setupSriovInterface(netns ns.NetNS, containerID, ifName string, ifInfo *Pod
 func ConfigureOVS(ctx context.Context, namespace, podName, hostIfaceName string,
 	ifInfo *PodInterfaceInfo, sandboxID string, podLister corev1listers.PodLister,
 	kclient kubernetes.Interface) error {
-	klog.Infof("ConfigureOVS: namespace: %s, podName: %s", namespace, podName)
+
 	ifaceID := util.GetIfaceId(namespace, podName)
 	initialPodUID := ifInfo.PodUID
 
+	ipStrs := make([]string, len(ifInfo.IPs))
+	for i, ip := range ifInfo.IPs {
+		ipStrs[i] = ip.String()
+	}
+
+	klog.Infof("ConfigureOVS: namespace: %s, podName: %s, SandboxID: %q, UID: %q, MAC: %s, IPs: %v",
+		namespace, podName, sandboxID, initialPodUID, ifInfo.MAC, ipStrs)
```

Comment:
`ifaceID` ?

Reply:
@msherif1234 that's just created from the namespace + name so I figured it was pointless to add because we have both pod + namespace in the message already. But we could if you like.

Reply:
true scratch that